pkg/common/constants.go

```diff
@@ -38,3 +41,11 @@ const DatabaseFlagUsage = "database in the catalog"
 const SchemaFlag = "schema"
 const SchemaFlagShort = "s"
 const SchemaFlagUsage = "schema in database on catalog"
+
+const BlueprintFlag = "blueprint"
+const BlueprintFlagShort = "b"
+const BlueprintFlagUsage = "fetch a blueprint data policy from a catalog or a processing platform"
+
```

Comment:
Why did we put this back in? Thought we'd removed the blueprint flag in the cli altogether?

Reply:
Yes, but we forgot one thing: if you want to get a Data Policy that is actually stored in the PACE DB, you need to provide a processing platform (as it's a required field). However, the current behavior results in fetching a blueprint policy from the processing platform if specified.